rentals.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'person'
+require_relative 'book'
+
+class Rentals
+  attr_accessor :date
+  attr_reader :person, :book
+
```

Comment:
- [ ] Please make sure all the variables `date, person, book` are readable and writable to work for the associations as expected,  To achieve this, You can add person and book variables in the accessor.